lms/templates/learner_dashboard/program_card.underscore

```diff
@@ -1,30 +1,49 @@
 
+<div class="text-section">
+    <h3 id="program-<%- id %>" class="title"><%- gettext(name) %></h3>
+    <div class="meta-info">
+        <div class="organization"><%- orgList %></div>
+        <div class="category">
+            <span class="category-text"><%- gettext(type) %></span>
+            <span class="xseries-icon" aria-hidden="true"></span>
+        </div>
+    </div>
+    <p class="certificate-status"><%= interpolate(
+        gettext('You have earned Verified Certificates in %(completed_courses)s of the %(total_courses)s courses so far.'),
+        {completed_courses: progress.total.completed, total_courses: progress.total.courses}, true
+    ) %></p>
+</div>
+<div class="progress">
+    <div class="bar complete" style="width:<%- progress.percentage.completed %>;"></div>
+    <div class="bar in-progress" style="width:<%- progress.percentage.in_progress %>;">
+        <span class="sr"><%= interpolate(
+            ngettext(
```

Comment:
A translation function that allows different string based on a number (for singular and plural).